sound/soc/codecs/jz4770.c

```diff
@@ -636,8 +640,8 @@ static void jz4770_codec_codec_init_regs(struct snd_soc_component *codec)
 	/* Disable ADC lrswap */
 	regmap_set_bits(regmap, JZ4770_CODEC_REG_CR_ADC, REG_CR_ADC_LRSWAP);
 
-	/* default to cap-less mode(0) */
-	regmap_clear_bits(regmap, JZ4770_CODEC_REG_CR_HP, REG_CR_HP_SB_HPCM);
+	/* default to cap-coupled mode(1) */
+	regmap_set_bits(regmap, JZ4770_CODEC_REG_CR_HP, REG_CR_HP_SB_HPCM);
```

Comment:
Changing the default to cap-coupled will break the radio on the GCW-Zero. I think you should instead add a DAPM widget (switch, probably), that can then be linked in devicetree as part of the headphones/speakers route.